ttyclock.c

```diff
@@ -174,7 +174,6 @@ update_hour(void)
      int ihour;
      char tmpstr[128];
 
-     ttyclock.lt = time(NULL);
```

Comment:
i think there's still a problem here: we remove the `lt` initialization which means `lt` will be undefined in the struct, which can lead to undefined behaviour (UB) which we never want. let's set it to *something*, even if we overwrite it later, it won't hurt, right?

Reply:
Btw. the reason to not have it here is because without some kind of ugly flag, update_hour doesn't know whether this is the first time the timestamp is fetched, in which case rounding it is generally prohibited because the initial display of time must show the non-rounded time at startup.